<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowTitle">
   <string/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="exceptPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>An unknown error has occurred.
Please report this error to software developer</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPlainTextEdit" name="ep_text"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mysqlCreatePage">
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>It seems that the database has not been formed!

Choose the procedure and click the next button.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QRadioButton" name="mcp_changeRadio">
         <property name="text">
          <string>Change database configuration</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QRadioButton" name="mcp_authRadio">
         <property name="text">
          <string>Create user with an authorized MySQL user</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QRadioButton" name="mcp_yourselfRadio">
         <property name="text">
          <string>Create new user yourself with MySQL values</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>276</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="mcp_nextButton">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mysqlServerPage">
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>MySQL Server not start.

Please start it and try again or close the application</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="msp_tryButton">
         <property name="text">
          <string>Try connection</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="msp_closeButton">
         <property name="text">
          <string>Close the application</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="authPage">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>92</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="ap_createButton">
         <property name="text">
          <string>Create User and Database</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="ap_backButton">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Authorized User Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ap_unameLine"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="ap_upassLine"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="confPage">
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>It seems that the software running for the first time.
Need to make some adjustments in order to use the software.
Please start by selecting your database from below</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Databases:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QRadioButton" name="cp_sqliteRadio">
            <property name="text">
             <string notr="true">SQLite</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="cp_mysqlRadio">
            <property name="text">
             <string notr="true">MySQL</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>232</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="cp_nextButton">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sqlitePage">
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Please specify the address of SQLite database file.</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="sp_defaultCheck">
            <property name="text">
             <string>Default Address</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="sp_addressButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Choose Address</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLineEdit" name="sp_addressLine">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>232</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="sp_backButton">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="sp_applyButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mysqlPage">
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="1" column="0">
        <widget class="QPushButton" name="mp_tryButton">
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Try Connection</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>228</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="mp_backButton">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="mp_applyButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>MySQL settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Server</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="mp_serverLine">
            <property name="text">
             <string notr="true">localhost</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>User Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="mp_unameLine">
            <property name="text">
             <string notr="true">bilge</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>User Pass</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="mp_upassLine">
            <property name="text">
             <string notr="true">123456</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="mp_defaultButton">
            <property name="text">
             <string>Defaults</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>DB Name</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="mp_dbLine">
            <property name="text">
             <string>bilgedb</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sp_defaultCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>sp_addressLine</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sp_defaultCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>sp_addressButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>msp_closeButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
